pyseir/inference/infer_rt.py

```diff
@@ -195,6 +195,9 @@ def apply_gaussian_smoothing(self, timeseries_type, plot=False):
             idx_start = 0
         else:
             idx_start = nonzeros[0]
+        if max(smoothed) < threshold:
```

Comment:
Can you give a bit more context around what the threshold is? 

Just to make sure I'm reading it right, this is saying if the max value in the entire smoothed array is less than the threshold, we skip the entire array.

I'm wondering if threshold could be renamed to make it clear what that threshold is.

Reply:
I updated the name and doc string to try to make it a bit clearer, but this is a very "pragmatic" addition. 

We're seeing a lot of noise from the inclusion of death data in our R(t) calculation whenever there are very few deaths (as in Alaska and Idaho, most recently). This parameter just gives us an easy binary way of filtering out (our use of) deaths for such cases.